player/lua/select.lua

```diff
@@ -28,9 +28,14 @@ end
 mp.add_forced_key_binding(nil, "select-playlist", function ()
     local playlist = {}
     local default_item
+    local show_title = mp.get_property_native("osd-show-playlist-titles")
 
     for i, entry in ipairs(mp.get_property_native("playlist")) do
-        playlist[i] = select(2, utils.split_path(entry.filename))
+        if entry.title and show_title then
```

Comment:
I was thinking that for fuzzy search, we might want to actually show both?

`<title> (<filename>)`

would satisfy both cases, when title is bogus or filename is bogus, without having to cycle `osd-show-playlist-titles`.